### PR TITLE
Tickets/dm 44739

### PR DESCRIPTION
**Pull request by teweitsai**
* Adapt the **PySide6**.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -1,7 +1,8 @@
-@Library('JenkinsShared')_
+@Library('JenkinsShared@tickets/DM-44739')_
```
Comment (wvreeven) [inline Jenkinsfile]: Please revert this before merging and after merging your JSL PR.

File: README.md [hunk 2]
```diff
@@ -11,9 +11,10 @@
 
 - numpy (install by `conda`)
 - pyyaml (install by `conda`)
-- pyside2 (install by `conda`)
-- qt5-qtbase-devel (install by `yum`)
-- xorg-x11-server-Xvfb (optional, install by `yum`)
+- pyside6 (install by `conda`)
+- qt6-charts (install by `conda`)
+- qt5-qtbase-devel (install by `dnf`)
```
Comment (wvreeven) [inline README.md:16]: Are you sure this and the next line are needed? When using conda all required system libraries get installed without the need for using dnf.

**Review by wvreeven — APPROVED** [review]
LGTM. Please see my comments before merging.

File: Jenkinsfile (hunk 1 above)
Comment (teweitsai) [reply Jenkinsfile]: Removed. Thanks!

File: README.md (hunk 2 above)
Comment (teweitsai) [reply README.md:16]: That was from the previous ticket when we experimented the PySide2. Let us keep this for now and test the minimum dependency with PySide6 when we are ready to remove the PySide2. As you know, the QT framework is a headache (at least, a little better than LabVIEW...) Thanks!

**Pull request merged**